pypesto/ensemble/ensemble.py

```diff
@@ -1253,3 +1269,65 @@ def calculate_cutoff(
 
     range = chi2.ppf(q=percentile / 100, df=df)
     return fval_opt + range
+
+
+def calculate_hpd(
+    result: Result,
+    burn_in: int,
```

Comment:
```suggestion
    burn_in: int = 0,
```
Just for the sake of having a default value here not requiring an input